openmc/statepoint.py

```diff
@@ -147,12 +138,9 @@ def __init__(self, filename, autolink=True):
                     vol = openmc.VolumeCalculation.from_hdf5(path_i)
                     self.add_volume_information(vol)
 
-    def close(self):
```

Comment:
h5py automatically closes a file when a `h5py.File` object goes out of scope. Thus, when a statepoint goes out of scope, the file will also close.